### PR TITLE
Session token support in SDK Object RPC calls

### PR DESCRIPTION
**Pull request by alexvanin**
Sort of draft, will be updated after v2.0.0-rc-1.

### CODE REVIEW

**Review by realloc — APPROVED** [review]

File: pkg/token/session.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		pubKey: key,
+	}, nil
+}
+
+func (s SessionToken) SessionKey() []byte {
+	return s.pubKey
+}
+
+func (s SessionToken) ID() []byte {
+	return s.id[:]
```
Comment (cthulhu-rider) [inline pkg/token/session.go:34]: I suggest `return s.id.Bytes()`

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/token/session.go (hunk 1 above)
Comment (alexvanin) [reply pkg/token/session.go:34]: `uuid.UUID` doesn't have `Bytes()` method, only `MarshalBinary()` that returns `([]byte, error)`

Comment (cthulhu-rider) [reply pkg/token/session.go:34]: Let's use `MarshalBinary` then, it never returns an error.

Comment (alexvanin) [reply pkg/token/session.go]: Done

**Pull request merged**